src/lib/opal_parser.c

```diff
@@ -161,25 +181,28 @@ int append_bytes(uint8_t *buf, size_t len, const uint8_t *src, int src_len)
 
 	/* data fits into opal short atom */
 	if (src_len <= 15) {
-		if (len >= src_len + 1) {
```

Comment:
please fix coding file for commenting and conditional block. for example:

/* comment 1*/
if (.....) {
    .....
/* comment 2*/
} else if (...) {
   .....
/* comment 3*/
} else {
   .....
}

Reply:
will fix it